config/playground/demo/observability/otel_demo/data-prepper.yaml

```diff
@@ -0,0 +1,151 @@
+# Copyright The OpenTelemetry Authors
+# SPDX-License-Identifier: Apache-2.0
+---
+# Data Prepper Deployment
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: data-prepper
+  namespace: otel-demo
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: data-prepper
+  template:
+    metadata:
+      labels:
+        app: data-prepper
+    spec:
+      containers:
+        - name: data-prepper
+          image: opensearchproject/data-prepper:2.6.1
+          ports:
+            - containerPort: 21890
+          volumeMounts:
+            - name: config-volume
+              mountPath: /usr/share/data-prepper/pipelines/pipelines.yaml
+              subPath: pipelines.yaml
+            - name: config-volume
+              mountPath: /usr/share/data-prepper/config/data-prepper-config.yaml
+              subPath: data-prepper-config.yaml
+      volumes:
+        - name: config-volume
+          configMap:
+            name: data-prepper-config
+      restartPolicy: Always
+---
+# Data Prepper config-map
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: data-prepper-config
+  namespace: otel-demo
+data:
+  pipelines.yaml: |
+    otel-logs-pipeline:
+      workers: 5
+      delay: 10
+      source:
+        otel_logs_source:
+          ssl: false
+      buffer:
+        bounded_blocking:
+      sink:
+        - opensearch:
+            hosts: ["https://opensearch-cluster-leader.default.svc:9200"]
+            username: "admin"
+            password: "admin"
```

Comment:
Does it make sense to change these credentials to be something like `CHANGE_ME_OS_ADMIN_USER` and `CHANGE_ME_OS_ADMIN_PWD`? That way it is clear to the user they need to change the credentials to what they have on their opensearch cluster?